packages/contracts/src/deprecated/interfaces/IERC20Ubiquity.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.3;
```

Comment:
we've agreed before to avoid floating and everything to `0.8.19;`

Reply:
I think it's better to keep the deprecated contracts untouched so that they were equal to their already deployed versions

Reply:
yeah i agree too